### ISSUE TITLE
Changeling headslugs are broken 

### ISSUE THREAD

**Issue by gallywix4** [tgstation/tgstation#56314]
## Testmerges: The following pull requests are currently test merged:
#55534: '/mob/living/proc/Life(delta_time)' by TemporalOroboros at commit 64722d2fea
#56061: 'Modernizes Admin Only Wizard Event, Department Revolt' by tralezab at commit 7d16921164

Reproduction:
Bug number 1:
Be a Changeling and use the skill last resort, you will realize your ling abilities still show up in the hud while before it didn't
Bug number 2:
Be a Changeling and grab the skill last resort
Grab a dead body and utilize the skill to become a headslug and lay a egg on said body
The timer for the revive will be longer then expected (on said round it was more then 40 minutes)
Bug number 3:
Repeat all steps for bug number 1
after that take control of another mob (on said round i was a ling that became a blob due to Midround antag)
When the changeling egg hatches you will be ghosted out of the body from said mob and get back to control the ling monkey

should be noted that after the third bug happend to me i instantly died and received a message about dying due to brain damage,while i can't say for certain if this is related to Bug 3 due to me not being able to use a private server to reproduce it at the time of writing i think it's impotant to point it out since i have no idea how that happend

**Comment by ATH1909** [comment]
Can you use those ling abilities from bug 1? What if you try to use last resort again?

**Comment by gallywix4** [comment]
I don't believe there is an affect for using the lings abilites from bug 1 since the headslug lacks the chemical reservers to spend on said abilities

**Comment by Sealed101** [comment]
1st point - still stands though it doesn't let you use the abilities since you're not a human mob. can't see it being too big of an issue though
https://github.com/tgstation/tgstation/blob/3ab970329867c340ba19c1caf4cc7559308ac317/code/modules/antagonists/changeling/changeling_power.dm#L107-L115
2nd point - unable to reproduce; perhaps a fluke of some kind when setting up the hatching time for the egg?
3rd point - still stands though the monkey is pretty much fine and doesn't die. this is indeed wonky and it's strange that an antag datum would remain on the mind after taking up a new role via ghost spawn or midround spawn or what have you
https://github.com/tgstation/tgstation/blob/3ab970329867c340ba19c1caf4cc7559308ac317/code/modules/antagonists/changeling/headslug_eggs.dm#L26-L49
